tests/data/data.json

```diff
@@ -1952,6 +1952,7 @@
     "$schema": "https://ils.rero.ch/schema/items/item-v0.0.1.json",
```

Comment:
Why we don't have items with type issue?

Reply:
because we do not have a lot of holdings of type journals (coming later). I create the items of type `issue` in the `holdings.cli.create_issues_from_holding`